pages/about/policies.html

```diff
@@ -287,7 +287,7 @@ <h1 class="ui header">
                     School Organisation
                 </h1>
                 <p>  
-                Preview Content goes here
+                <strong><em>School Organisation Policy</em></strong> "Mount Sion was founded by Edmund Rice in 1802. It was the first school founded by the Christian Brothers. The school has a proud tradition of educational service in Waterford and the surrounding areas...
```

Comment:
also @denispop9 I think this should be included as part of another pr?

Reply:
I'm going to do that right now:)